templates/zerver/help/logging-in.md

```diff
@@ -13,27 +13,34 @@ how you signed up. E.g. if you originally signed up using your Google
 account, you can later log in using GitHub, as long as your Google account
 and GitHub account use the same email address.
 
-
 ### Log in to a Zulip organization for the first time
```

Comment:
Please update the headers in this article so that the top ones are `##` level, and the next ones are `###`. We have been cleaning up all the places where `##` headers are skipped.